component/newstorage/formattedstore/formattedstore.go

```diff
@@ -100,6 +140,16 @@ func (f *FormattedProvider) SetStoreConfig(name string, config newstorage.StoreC
 // The store must be created prior to calling this method.
 // If the store cannot be found, then an error wrapping ErrStoreNotFound will be returned.
 func (f *FormattedProvider) GetStoreConfig(name string) (newstorage.StoreConfiguration, error) {
+	if f.cacheProvider != nil {
+		config, err := f.cacheProvider.GetStoreConfig(name)
+		if err == nil {
+			return config, nil
+		}
+
+		logger.Debugf(`cache miss: failed to get store config for store "%s" from cache storage provider: %s`,
```

Comment:
no need for the message to look like an error (it would confuse ops), just a simple `cache miss: fetching data from real store` suffice.

also, you can check the type of error returned by the cache (if err != notFoundErr then it's a valid cache miss error, otherwise, print a real error?)

Reply:
You're right. FIxed!